proposals/containerd.adoc

```diff
@@ -0,0 +1,55 @@
+== containerd Proposal
+
+*Name of project:* containerd
+
+*Description*
+
+containerd is an industry-standard container runtime with an emphasis on simplicity, robustness and portability. It is available as a daemon for Linux and Windows, which can manage the complete container lifecycle of its host system: image transfer and storage, container execution and supervision, and low-level storage, etc..
+
+containerd is designed to be embedded into a larger system, rather than being used directly by developers or end-users. containerd will serve as a core container runtime for the CNCF ecosystem.
+
+*Sponsor / Advisor from TOC*: Brian Grant
+
+*Unique Identifier*: containerd
+
+*License*: Apache License v2.0
+
+*Source control repositories*: https://github.com/docker/containerd
```

Comment:
The intention is to move to a github org that 1) is not Docker-branded and 2) is controlled by Limix Foundation.

github.com/containerd is an obvious possibility. Full disclosure we are considering donating more projects, currently grouped under the Docker project. If that is confirmed then we might keep these repos under a common org to preserve the existing workflow, tooling and integrations. Either way the org will be non-Docker and lf-controlled.

With that guarantee in mind, can we request a few weeks grace period before we lock down the exact name of the final org? It would be much appreciated.